src/components/SearchGames.js

```diff
@@ -152,9 +152,11 @@ const SearchGames = ({ userGames, addGame, removeGame }) => {
         <Text style={{ fontSize: 20, color: "#3AE456" }}>Search</Text>
       </Pressable>
       {searching && (
-        <View>
-          <Text style={styles.searchingMessage}>Searching...</Text>
-        </View>
+          <ActivityIndicator
+            style={{ marginTop: 150 }}
+            size="large"
+            color="#3AE456"
+          />
```

Comment:
The only thing I want don't love about this is that it slightly pushes up the inputs when it renders. I've tried a few different styling options but I can't figure out how to avoid this. Maybe one of you can help me figure this out.

Reply:
I noticed that after you mentioned it, but didn't before I read this. I tried to fix it by adding a container around all the search views, and giving it height but it was being finicky. The text search container was gifting squished and I couldn't get it to be the right hight. I figured it may not be something I should spend too much time on rn.